INSTALL.md

```diff
@@ -0,0 +1,25 @@
+# Local Installation
+
+The following is instructions on how to install the Core Framework on your local machine. This will allow you to develop or edit applications for the Core Framework. These are not instructions for a production-ready deployment.
+
+These instructions should work on Windows, MacOS and Linux.
+
+First, install Docker by following the [docker desktop instructions](https://www.docker.com/products/docker-desktop) for your operating system. If you're on Linux you'll need to work out the right approach for your specific distribution, including install docker-compose separately.
+
+Next, download the [docker-compose.yml](./docker-compose.yml) file from GitHub and place it in a directory of your choosing. If you are unfamilar with terminals (see below) we recommend you place it in your home directory. (`/home/USERNAME` in MacOS, usually `C:\Users\USERNAME` in Windows).
```

Comment:
Edited since it wasn't clear what "see below" was directing to, or why it was helpful to see the terminal instructions at that point.

```suggestion
Next, download the [docker-compose.yml](./docker-compose.yml) file from GitHub and place it in a directory of your choosing. If you are unfamilar with terminals we recommend you place it in your home directory. (`/home/USERNAME` in MacOS, usually `C:\Users\USERNAME` in Windows).
```